packages/elements/src/photon-medication-search/photon-medication-search-component.tsx

```diff
@@ -257,10 +257,11 @@ const Component = (props: ComponentProps) => {
   const tryLoadTreatmentOptions = async (searchTerm: string) => {
     setLoadingTreatmentOptions(true);
 
-    const treatmentOptions = searchTerm
-      ? await loadTreatmentOptions(client!.sdk.apolloClinical, searchTerm)
-      : // Set treatment options to empty array if search term is empty
-        [];
+    const treatmentOptions =
+      searchTerm?.length >= 3 // 3 min chars for smaller responses
+        ? await loadTreatmentOptions(client!.sdk.apolloClinical, searchTerm)
+        : // Set treatment options to empty array if search term is empty
+          [];
```

Comment:
Nit, I would differentiate between `[]` which means "no results" and `null` which can mean "not searching"

Also will make it easier to solve the fact that it first shows "No Data"

Reply:
that's good feedback, need to think more about this because it affects the 3 dropdowns, moving forward with this small change for the moment.